docs/design/aws_under_the_hood.md

```diff
@@ -0,0 +1,271 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/design/aws_under_the_hood.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Peeking under the hood of kubernetes on AWS
+
+We encourage you to use kube-up (or CloudFormation) to create a cluster.  But
+it is useful to know what is being created: for curiosity, to understand any
+problems that may arise, or if you have to create things manually because the
+scripts are unsuitable for any reason.  We don't recommend manual configuration
+(please file an issue and let us know what's missing if there's something you
+need) but sometimes it is the only option.
+
+This document sets out to document how kubernetes on AWS maps to AWS objects.
+Familiarity with AWS is assumed.
+
+### Top-level
+
+Kubernetes consists of a single master node, and a collection of minion nodes.
```

Comment:
Question: Are we able to avoid using the term "minion" in this doc? The correct term now is "node" but i see that there is still other references in the code like: `kubernetes-minion-<clusterid>`.